src/engine/strat_engine/filesystem.rs

```diff
@@ -109,6 +161,31 @@ impl Recordable<FilesystemSave> for StratFilesystem {
     }
 }
 
+/// Return total bytes, total bytes used
+pub fn fs_usage(mount_point: &PathBuf) -> EngineResult<(Bytes, Bytes)> {
+    let mut stat = Statvfs::default();
+    try!(statvfs(mount_point, &mut stat));
+    let total_bytes = stat.f_blocks * stat.f_bsize;
+    Ok((Bytes(total_bytes), Bytes(total_bytes - stat.f_bfree * stat.f_bsize)))
+}
+
+/// Use the xfs_growfs command to expand a filesystem mounted at the given
+/// mount point.
+/// TODO should we log failures here?
```

Comment:
You can leave this out...it's a question that can be asked about most points in our code.